filter-median.h

```diff
@@ -0,0 +1,23 @@
+#ifndef __FILTER_MEDIAN__
+#define __FILTER_MEDIAN__
+
+#include <algorithm>
+
+int filterData()
+{
```

Comment:
filterMedian()

Reply:
For discussion: rename this function or create separate namespace(Filter for example). and implement there different filtering algorithms?

Reply:
what is better (faster) for you?
rename is minimum requirement